ci/deploy.yml

```diff
@@ -195,6 +195,27 @@ release core unix debug deploy:
   tags:
     - deploy
 
+release core unix debug arm deploy:
+  stage: deploy
+  variables:
+    DEPLOY_PATH: "${DEPLOY_BASE_DIR}/releases/emulators"
+    GIT_SUBMODULE_STRATEGY: "none"
+  before_script: []  # no poetry
+  needs:
+    - core unix frozen debug build arm
+  script:
+    - export VERSION=$(./tools/version.sh core/embed/firmware/version.h)
+    - DEST="$DEPLOY_PATH/trezor-emu-core-arm-v$VERSION"
+    - echo "Deploying to $DEST"
+    - nix-shell -p patchelf --run "patchelf --set-interpreter /lib/ld-linux-aarch64.so.1 core/build/unix/trezor-emu-core"
+    - rsync --delete -va core/build/unix/trezor-emu-core "$DEST"
+  only:
+    - /^core\//
+  except:
+    - branches  # run for tags only
+  tags:
+    - deploy
```

Comment:
Sorry for delaying the PR but is there a reason not to make it part of `release core unix debug deploy`? We can _need_ both of the builds and deploy at the same time. Yes, it will break if one of the builds fails, but I think that is acceptable. We should try to make the jobs compact.

Reply:
Both artifacts have the same folder structure and naming, not sure if it won't get overwritten. My first idea was to add it there, but I think it won't work like that.

Reply:
Oh, I see. Hmm, let's rename it to `*-arm` after the build here? 
https://github.com/trezor/trezor-firmware/pull/2025/files#diff-fad8dced0db200747490daa0aa2fb66c1d30378b9cf5ba032ae96e2e5d8825c1R168

Reply:
I tried to do it like that but I'm not sure if its ok that it will overwrite other files but I think that this way we should have both emualtors and we should be able to deploy it with one job.

Reply:
What other files it would overwrite?

Reply:
For the core, we have all of the contents of `core/build/unix` but that is not needed for emulator or tenv. Should be fine 👍